recipes/omero-py/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "omero-py" %}
+{% set version = "5.10.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/omero-py-{{ version }}.tar.gz
+  sha256: 4fbf7288586093d45e8f3a661ddf46143f38231017fada39c4e1c26725b34e33
+
+build:
+  entry_points:
+    - omero=omero.main:main
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - appdirs
+    - future
+    - numpy
+    - pillow
+    - python
+    - pywin32  # [win]
```

Comment:
This is the only package which means this can't be noarch :disappointed: 

Nothing can be done about this now, but keep an eye on https://github.com/conda-forge/pywin32-feedstock/pull/37 (@bollwyvl)

Reply:
Might be able to use this now: https://github.com/conda-forge/pywin32-on-windows-feedstock